oda_api/api.py

```diff
@@ -816,7 +816,7 @@ def get_product(self,
             instrument=instrument, product_type=product), cookies=self.cookies)
 
         if res.status_code != 200:
-            warnings.warn(
```

Comment:
although we did not use `warnings` much, it's not the same as `logging.warning`. See [here](https://docs.python.org/3/howto/logging.html). 

```
`warnings.warn()` in library code if the issue is avoidable and the client application should be modified to eliminate the warning

`logging.warning()` if there is nothing the client application can do about the situation, but the event should still be noted
```

In this case, we actually want the client to do something: adapt parameters. So, following the python convention, it should be `warnings.warn`